cmd/vela-slack/main.go

```diff
@@ -17,6 +18,11 @@ import (
 	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli/v2"
 
+	"crypto/tls"
+	"crypto/x509"
+
+	"gopkg.in/ldap.v2"
```

Comment:
i see there is a `ldap.v3` (https://pkg.go.dev/gopkg.in/ldap.v3) - should we update to that?

Reply:
done!